docs/commands/output.md

```diff
@@ -12,8 +12,8 @@ floyd output [OPTIONS] ID
 | NAME or ID |      | Name or ID of your job. |
 
 ### Description
-Most jobs generate output. Any output that needs to be retained after the job is finished should be send to `/output` path.
-This is the only path Floyd will preserve. The output command gives the url to access this output. This command by default opens the
+Most jobs generate output. Any output that needs to be retained after the job is finished should be saved inside working directory (`/floyd/home`).
+This is the only path Floyd will preserve across Jobs and Workspaces. The output command gives the url to access this output. This command by default opens the
```

Comment:
s/Floyd/FloydHub/

I don't think 'across Jobs and workspaces' is relevant here since this is explaining the command jobs. You can may be say 'across all jobs'